python/paddle/fluid/tests/unittests/test_var_base.py

```diff
@@ -652,6 +652,59 @@ def _test_slice_for_tensor_attr(self):
             np.array_equal(local_out[15], tensor_array[::-1, ::-1, ::-1]))
         self.assertTrue(np.array_equal(local_out[16], tensor_array[-4:4]))
 
+    def _test_for_getitem_ellipsis_index(self):
+        shape = (64, 3, 5, 256)
+        np_fp32_value = np.random.random(shape).astype('float32')
+        np_int_value = np.random.randint(1, 100, shape)
+
+        var_fp32 = paddle.to_tensor(np_fp32_value)
+        var_int = paddle.to_tensor(np_int_value)
+
+        # test for float32
+        var = [
```

Comment:
duplicate code? put the duplicate code in another func?

Reply:
Done!